include/capstone/x86.h

```diff
@@ -300,23 +308,6 @@ typedef struct cs_x86 {
 	// Address size, which can be overrided with above prefix[5].
 	uint8_t addr_size;
 
-	// ModR/M byte
-	uint8_t modrm;
```

Comment:
I thought it made more sense to have all the related info in one place. Easy to undo if you want to stay backwards compatible, though.